packages/trie/src/db/checkpoint.ts

```diff
@@ -114,19 +156,27 @@ export class CheckpointDB implements DB {
   /**
    * @inheritDoc
    */
-  async put(key: Uint8Array, val: Uint8Array): Promise<void> {
+  async put(key: Uint8Array, value: Uint8Array): Promise<void> {
+    const keyHex = bytesToHex(key)
+    if (this._cache !== undefined) {
+      this._cache.set(keyHex, value)
+      this._stats.cache.writes += 1
+    }
     if (this.hasCheckpoints()) {
       // put value in cache
```

Comment:
What is the difference between this cache and the one above?  I guess maybe we should refer to this as the "diffCache" or something like that to match what we do in the `statemanager` cache comments.

Reply:
Good point, have updated mainly (only) in comments, since we generally had no code cache references in this file before.